templates/code_generation_swift_file.mustache

```diff
@@ -0,0 +1,81 @@
+import UIKit
+
+protocol LocalisationProvider {
```

Comment:
Elsewhere we use American spelling (`Localizable` as filename). We should pick one or the other?